src/NServiceBus.Testing/NSB.Testing.Fakes/TestableBehaviorContext.cs

```diff
@@ -22,6 +23,11 @@ public abstract partial class TestableBehaviorContext : IBehaviorContext
         /// </summary>
         public ContextBag Extensions { get; set; } = new ContextBag();
 
+        /// <summary>
+        /// A <see cref="CancellationToken"/> to observe during message processing. Use this to verify behavior when message processing is cancelled.
+        /// </summary>
+        public CancellationToken CancellationToken { get; set; } = CancellationToken.None;
```

Comment:
The assignment is [redundant](https://github.com/dotnet/runtime/blob/master/src/libraries/System.Private.CoreLib/src/System/Threading/CancellationToken.cs#L44).

```suggestion
        public CancellationToken CancellationToken { get; set; }
```